### ISSUE TITLE
a11y: The "+" on Tabs on the bottom should not be read (aria-hidden)

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#118]
## Overview
From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

### Issue 8: Tab list in the bottom has different tags
ARIA attributes do not conform to valid values (4.1.1, 1.3.1, 4.1.2)

### Reference to standards
4.1.1 Parsing: In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features. (Level A)
1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. (Level A)
4.1.2 Name, Role, Value: For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies. (Level A)
### Suggestion

Ensure the value inside each attribute is spelled correctly and corresponds to a valid value (For example, aria- hidden="true" would pass, but aria-hidden="rtue" would fail, as would aria-hidden="pizza".)

## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df676574efd7dc06228ea/72c7fe50e074a1f273f194e4b3957355/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df676574efd7dc06228ea/1c5d01e9960e038714e1ac66d6570e7b/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df676574efd7dc06228ea/6cf6d24661a4295dca302faac098bee2/image.png)
